packages/components/Button/src/Button.types.ts

```diff
@@ -166,14 +166,23 @@ export interface ButtonProps extends ButtonCoreProps {
   loading?: boolean;
 }
 
+interface ButtonState {
+  measuredHeight?: number;
+  measuredWidth?: number;
+
+  // win32 only. Whether the component should use a tone-tone focus border instead of single-tone
+  shouldUseTwoToneFocusBorder?: boolean;
+}
+
 export interface ButtonInfo {
   props: ButtonProps & React.ComponentPropsWithRef<any>;
-  state: PressableState;
+  state: PressableState & ButtonState;
```

Comment:
nit: should ButtonState just extend PressableState? There was a pattern I followed when converting other views to `usePressableState`, lemme look it up..

Reply:
Yeah, that's what we did for Checkbox and Switch

Reply:
Sure, can do.